kimchi/src/tests/lookup.rs

```diff
@@ -157,5 +173,16 @@ fn test_runtime_table() {
     print_witness(&witness, 0, 20);
 
     // run test
-    TestFramework::run_test_lookups(gates, witness, &[], vec![], Some(runtime_tables));
+    TestFramework::default()
+        .gates(gates)
+        .witness(witness)
+        .runtime_tables_cfg(runtime_tables_cfg)
+        .runtime_tables(runtime_tables)
```

Comment:
Nit: `runtime_tables` should probably come after `setup`, since it's not required for the setup.